submissions/fabunmiTolulope/dom/signUp.js

```diff
@@ -0,0 +1,77 @@
+let submitForm = document.getElementById('validateForm');
+let firstName = document.getElementById('first-name');
+let lastName = document.getElementById('last-name');
+let phoneNumber = document.getElementById('phone-number');
+let email = document.getElementById('email');
+let password = document.getElementById('password');
+let confirmPassword = document.getElementById('confirm-password');
+
+function validation(field, Regex, errorMessage1, errorMessage2, event) {
+    if (field.value === "") {
+      field.nextElementSibling.innerHTML = errorMessage1;
+      event.preventDefault();
+      return false;
+    }
+    if (!field.value.match(Regex)) {
+      field.nextElementSibling.innerHTML = errorMessage2;
+      event.preventDefault();
+      return false;
+    } else {
+      field.nextElementSibling.innerHTML = "";
+      return true;
+    }
+  }
+function validateForm(event) {
+    const nameRegex = /^[_A-z0-9]*((-|\s)*[_A-z0-9])*$/g;
```

Comment:
declare all these variables outside of the function

Reply:
@adaugochi I have declared the variables outside of the function